unreal_qt_stylesheet/main.py

```diff
@@ -0,0 +1,61 @@
+# import sys
+import os
+
+
+try:
+    from PySide2 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PyQt5 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PySide6 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PyQt6 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+# from Qt import QtWidgets, QtCore, QtGui
+# from Qt import _loadUi
+from pathlib import Path
+
+
+# MODULE_PATH = os.path.dirname(os.path.abspath(__file__))
+# UI_PATH = os.path.join(MODULE_PATH, 'ui', 'progress.ui')
+# QSS_PATH = os.path.join(MODULE_PATH, 'ue.qss')
+MODULE_PATH = Path(os.path.abspath(__file__)).parent
+QSS_PATH = MODULE_PATH / 'ue.qss'
+
+
+def setup():
+    """ Apply the Unreal dark stylesheet to the current QApplication """
```

Comment:
just a style issue: I like the docstring written in this way:
```
""" 
Apply the Unreal dark stylesheet to the current QApplication 
"""
```

Reply:
ok will change it